### ISSUE TITLE
Mirror's Edge [NPUB30838]: Consistent freezing

### ISSUE THREAD

**Issue by Shoegzer** [RPCS3/rpcs3#11851]
## Quick summary
The game runs fine until the player reaches a specific point, then it freezes consistently.

## Details
In Chapter 6, the "Infiltrate Pirandello Kruger security" mission, right after the first use of the sniper rifle, the player ziplines to a far building.  Once the player goes through the door and attempts to jump from a stack of boxes to a metal railing, the game freezes.  This was tested three times and it freezes in the exact spot each time.  The following message appears in the log:

`F {PPU[0x100000b] Thread (RenderingThread) [0x00d6037c]} VM: Access violation writing location 0x0 (unmapped memory)`

Default settings are used.
 
### Test Information:
* OS: Linux Mint 20.3 / kernel 5.13
* CPU: Intel i7-10870H
* GPU: Nvidia GTX 3070
* Driver version: 510.60
* Log: [RPCS3.log](https://www.sendspace.com/file/diz8vy)

### Additional Information:
Issues #11695, #11821 and #11827 also report freezing with an "unmapped memory" exception, and all were reported in the last 30 days.  While they may be related, each report the exception from other threads than the "Rendering Thread" observed here.

**Comment by YuriNator557** [comment]
Link to log is invalid, please either upload it to GitHub or put it somewhere more permanent like Google Drive

**Comment by Megamouse** [comment]
Is this still an issue?

**Comment by Shoegzer** [comment]
I can't confirm as I don't have this game anymore.  However, someone who does could use the save I posted to do so.  Hopefully somoene else can help, as I recall this seemed quite a unique problem and tough to track down.

**Comment by digant73** [comment]
@Shoegzer is that still an issue?

**Comment by Shoegzer** [comment]
I can't confirm as I don't have this game anymore

**Comment by Triticum0** [comment]
There was full play-though on youtube of the game on the emulator https://www.youtube.com/watch?v=Fpe_TyIniJU after this issue was posted. ver 0.0.23-13931
So unless your going testing it with your hardware config you might as well close this issue.

**Comment by Shoegzer** [comment]
This is inconclusive - at [this](https://youtu.be/Fpe_TyIniJU?t=5176) point the video you linked to, which is at the section that crashes, you will notice a break in the video, as part of the gameplay was cut.  After the cut you will notice that the sniper rifle is not equipped and is not used. 

So what happened?

I believe this may actually be two attempts, the first attempt (showing before the cut) involved the sniper rifle, as you can see, but this crashed RPCS3.  So the crash was cut from the video and a second attempt (showing after the cut) did not involve the sniper rifle at all, to bypass the crash.  You can tell because the agents on the ground appear to still be alive as you can hear them shooting at the player.  Thus, entering the building from the upper-left zipline did not result in the crash once in the building, as shown.

Just a theory of course, again since I don't have the game I'll defer to others to confirm.

**Comment by Triticum0** [comment]
Yeah fair point, didn't actually check the video as didn't know where it crashed. Latest and only video after the reported so someone with the game would need to retest. Thank for you imput.

**Comment by alexconsole04** [comment]
The save provided only goes up to Chapter 5.

**Comment by Shoegzer** [comment]
I recall having saved at the closest point to where the issue occurs, but if that's chapter 5 then it's really not that far from it anyway.

**Comment by alexconsole04** [comment]
I don't exactly know what specific spot I'm supposed to crash, but game works fine I guess.
<img width="2560" height="1392" alt="Image" src="https://github.com/user-attachments/assets/e1fac0a5-f289-443b-bbc4-e739885d8aa5" />
<img width="2560" height="1392" alt="Image" src="https://github.com/user-attachments/assets/7799666e-ddac-464a-b3d3-1840b81e1721" />

**Comment by Shoegzer** [comment]
Did you trigger the conditions that cause the freezing (used sniper rifle just before entering the building through the red door on the ground?)  Otherwise, the title didn't freeze for me either.

**Comment by alexconsole04** [comment]
Yes, didn't crash.

**Comment by Shoegzer** [comment]
May need to close this then.  Can you post your specs (OS, CPU, GPU, Driver)?

**Comment by alexconsole04** [comment]
11400f
RTX 2060 596.36
Windows 11

**Comment by Shoegzer** [comment]
Thanks. Can you test from a Linux system?  Ideally would be good to have at least one confirmation from a linux test before closing as this may be OS-specific.
